### ISSUE TITLE
AS

### ISSUE THREAD

**Issue by AScustomWorks** [VCVRack/library#393]
Small update to fix some svg file names and custom color for vu meter leds background. Let's see if I got this right:
```
{
  "name": "AS",
  "author": "Alfredo Santamaria",
  "license": "MIT",
  "manualUrl": "https://github.com/AScustomWorks/AS/blob/master/README.md",
  "sourceUrl": "https://github.com/AScustomWorks/AS",
  "donateUrl": "https://www.paypal.me/frederius/",
  "latestVersion": "0.6.0.1",
  "status": "available"
}
```

**Comment by ZuluEchoRomeOscar** [comment]
Rack no longer sees AS plugin [Linux].
log.txt reports 
`[0.029 warn] Failed to load library /home/zero/.Rack/plugins/AS/plugin.so: /home/zero/.Rack/plugins/AS/plugin.so: undefined symbol: _ZN4rack9SVGSwitch4stepEv`

**Comment by AScustomWorks** [comment]
Another small update (0.6.0.2), BPM clock code fix, VCA internal name fix:
```
{
  "name": "AS",
  "author": "Alfredo Santamaria",
  "license": "MIT",
  "manualUrl": "https://github.com/AScustomWorks/AS/blob/master/README.md",
  "sourceUrl": "https://github.com/AScustomWorks/AS",
  "donateUrl": "https://www.paypal.me/frederius/",
  "latestVersion": "0.6.0.2",
  "status": "available"
}
```

**Comment by AndrewBelt** [comment]
Can I use you as a quick test? Since you created this issue, can you close it and reopen it to make sure you can do that? Afterwards, I'll close the issue, and then you check if you can reopen it again.

**Comment by AScustomWorks** [comment]
done!

**Comment by AndrewBelt** [comment]
Okay, now try to reopen.

**Comment by AScustomWorks** [comment]
The reopen button is gone, can't do it

**Comment by AScustomWorks** [comment]
After you reopened, I can see the close and comment button

**Comment by AndrewBelt** [comment]
Oh well, I was hoping developers could reopen themselves, but I suppose GitHub disables that for good reason. Since you can comment though, I'll recommend to plugin developers to just make a comment in their closed issue, someone will reopen it, and then someone on the Library Team will handle the request.

**Comment by mdemanett** [comment]
@AndrewBelt this would be good to go for review/build, if x.y.z.w versions will work (0.6.0.2 here)?

**Comment by AndrewBelt** [comment]
Yeah, that's fine, although I don't know why plugin developers do it. It could be `0.6.bicycle`, as long as it starts with `0.6.`

**Comment by phdsg** [comment]
updated: https://github.com/VCVRack/community/commit/37a08aba782e3f75397d6156e2e89fc62adcb30a

**Comment by AndrewBelt** [comment]
Removed repo submodule because libsamplerate doesn't build on my Mac build system yet. This should be an easy fix using @jeremywen's methodhttps://github.com/VCVRack/Fundamental/issues/38#issuecomment-377720447

**Comment by AScustomWorks** [comment]
Agree, but I don't have the slightest idea about where/how to make those changes, just followed your example  on Fundamental.

**Comment by AScustomWorks** [comment]
according to [45f6f7e](https://github.com/VCVRack/Fundamental/commit/45f6f7e85db4d19cbd78bf01d265ca94647fbc02)  my makefile should be ok now:

{
  "name": "AS",
  "author": "Alfredo Santamaria",
  "license": "MIT",
  "manualUrl": "https://github.com/AScustomWorks/AS/blob/master/README.md",
  "sourceUrl": "https://github.com/AScustomWorks/AS",
  "donateUrl": "https://www.paypal.me/frederius/",
  "latestVersion": "0.6.0.2",
  "status": "available"
}

**Comment by AScustomWorks** [comment]
Just curious, is there something still wrong? or maybe it just got forgotten?

**Comment by AScustomWorks** [comment]
Just updated to AS to v0.6.1, with a new module addition and another module update:
```
{
  "name": "AS",
  "author": "Alfredo Santamaria",
  "license": "MIT",
  "manualUrl": "https://github.com/AScustomWorks/AS/blob/master/README.md",
  "sourceUrl": "https://github.com/AScustomWorks/AS",
  "donateUrl": "https://www.paypal.me/frederius/",
  "latestVersion": "0.6.1",
  "status": "available"
}
```

**Comment by AndrewBelt** [comment]
The Review Team still hasn't updated the repo to fix the libsamplerate build.

**Comment by phdsg** [comment]
did i not? thought at least the 0.6.0.2 was ready...
@AndrewBelt did it fail during pull or make? or did i miss something?

**Comment by AScustomWorks** [comment]
Made a small fix to BPM Clock module, latest version is now 0.6.2:
```
{
  "name": "AS",
  "author": "Alfredo Santamaria",
  "license": "MIT",
  "manualUrl": "https://github.com/AScustomWorks/AS/blob/master/README.md",
  "sourceUrl": "https://github.com/AScustomWorks/AS",
  "donateUrl": "https://www.paypal.me/frederius/",
  "latestVersion": "0.6.2",
  "status": "available"
}
```

**Comment by AScustomWorks** [comment]
One last update before the next PM build I hope, latest version is now [0.6.3](https://github.com/AScustomWorks/AS/):
```
{
  "name": "AS",
  "author": "Alfredo Santamaria",
  "license": "MIT",
  "manualUrl": "https://github.com/AScustomWorks/AS/blob/master/README.md",
  "sourceUrl": "https://github.com/AScustomWorks/AS",
  "donateUrl": "https://www.paypal.me/frederius/",
  "latestVersion": "0.6.3",
  "status": "available"
}
```

**Comment by mdemanett** [comment]
Updated.  Checked the build -- works for me (on mac).

**Comment by phdsg** [comment]
@AndrewBelt looks like this wasn't this updated in the last cycle. still something wrong?

**Comment by AScustomWorks** [comment]
Indeed, v0.63 still not available on the plugin manager. Do I need to change something?

**Comment by AndrewBelt** [comment]
libsamplerate can't build on a mingw cross compiler on Linux, because the `./configure` step needs to execute built binaries. A fix is to adopt the libsamplerate .cpp files into your own build system and not use the autoconf scripts.

**Comment by AScustomWorks** [comment]
Thanks Andrew,  got rid of libsamplerate and using rack's own samplerate.hpp now so this issue should be resolved finally ;)

latest version is now [0.6.4](https://github.com/AScustomWorks/AS/):
```
{
  "name": "AS",
  "author": "Alfredo Santamaria",
  "license": "MIT",
  "manualUrl": "https://github.com/AScustomWorks/AS/blob/master/README.md",
  "sourceUrl": "https://github.com/AScustomWorks/AS",
  "donateUrl": "https://www.paypal.me/frederius/",
  "latestVersion": "0.6.4",
  "status": "available"
}
```

**Comment by AScustomWorks** [comment]
Now updated to [v.0.6.5](https://github.com/AScustomWorks/AS/)

**Comment by phdsg** [comment]
Updated build.

**Comment by AScustomWorks** [comment]
New module and couple of modifications on two modules. Now updated to [v0.6.6](https://github.com/AScustomWorks/AS)

**Comment by AndrewBelt** [comment]
Updated builds

**Comment by AScustomWorks** [comment]
3 new modules and a small add on to BPM clock. Now updated to [v0.6.7](https://github.com/AScustomWorks/AS/)

**Comment by AScustomWorks** [comment]
Small fixes to 3 modules + new feature added to BPM Calc module. Now updated to [v0.6.8](https://github.com/AScustomWorks/AS/)

**Comment by AndrewBelt** [comment]
Updated build

**Comment by AScustomWorks** [comment]
Fixed some code/added extras on a couple of modules, added 2 new modules. Now updated to [v0.6.9](https://github.com/AScustomWorks/AS)

**Comment by AndrewBelt** [comment]
Updated build

**Comment by AndrewBelt** [comment]
Updated build

**Comment by AScustomWorks** [comment]
Added 1 new module (slimmed down BPM Delay/MS calc module). Now updated to [V 0.6.11](url
https://github.com/AScustomWorks/AS)

**Comment by AndrewBelt** [comment]
Updated build

**Comment by AScustomWorks** [comment]
Small update:
Update on Triggers modules to use pulses.
Module name graphic update on BPM Delay/MS calc
Now updated to [v 0.6.12](https://github.com/AScustomWorks/AS)

**Comment by AndrewBelt** [comment]
Built updates (yesterday)

**Comment by AScustomWorks** [comment]
Hi everyone!, Small update:
Multi 2x5 module behavior update
2 and 4 channel mixer new modules
Now updated to [v 0.6.13](https://github.com/AScustomWorks/AS)

**Comment by AndrewBelt** [comment]
Updated build

**Comment by AScustomWorks** [comment]
Now updated to [V1](https://github.com/AScustomWorks/AS)

**Comment by cschol** [comment]
Invalid tags in `plugin.json`: https://github.com/AScustomWorks/AS/issues/41

**Comment by cschol** [comment]
Plugin crashes module browser: https://github.com/AScustomWorks/AS/issues/42

**Comment by cschol** [comment]
Plugin still crashes in other instances. See issue #42 for updates.

**Comment by AScustomWorks** [comment]
Hi, now updated to [V 1.0.1](https://github.com/AScustomWorks/AS)

Fixed Slug names on TinySine and TinySawish(LowerCase error)
SignalDelay snd port with pre/post delay switch
Small graphic refresh/fixes, output ports on every module are now gold colored to make easier to find them

**Comment by AScustomWorks** [comment]
Hi everyone! the plugin is updated to [V 1.0.2](https://github.com/AScustomWorks/AS) now

More graphic updates.
Now every knob tooltip shows the proper parameter value according to it's function.
BPM Clock got the BPM range switch removed to work ok with the tooltips, 30-300 BPM is now the default range.

**Comment by AScustomWorks** [comment]
Hi everyone! the plugin is updated to [V 1.0.3](https://github.com/AScustomWorks/AS)now.

Updates:
BPM Calc and BPM Calc 2: small code optimization.
Delay Plus Stereo FX: Now when the Feedback or Color channels are linked, the right knob reflects the setting from the left knob.
Super Drive FX Stereo module added
WaveShaper module: CV inputs to control shape and scale added, wave mod input added.
WaveShaper FX Stereo module added, same features as WaveShaper, but stereo.

**Comment by AScustomWorks** [comment]
Hi everyone! the plugin is updated to [V 1.0.4](https://github.com/AScustomWorks/AS) now.
Update: fixed the "disabled" property on the plugin.json file

**Comment by AScustomWorks** [comment]
Hi, AS plugin is ready for [V2](https://github.com/AScustomWorks/AS/tree/V2)
Also made an update for [V1](https://github.com/AScustomWorks/AS/tree/V1.05) to v1.05(Phaser bleed bug fixed - ZCV2T polyphony update )just incase you are still updating the V1 plugins.
Thanks!

**Comment by AndrewBelt** [comment]
Reopening for v2

**Comment by AScustomWorks** [comment]
Hi, made a small update to fix a crash with TriggersMKIII module(didn't noticed Rack textfield had changed for V2)
Also added clear buffer switches to DelayPlus Stereo FX module, some graphics updates
[https://github.com/AScustomWorks/AS/commit/abd5b5cde20f21f4ba6e1cb78d6d8097e96471dc](https://github.com/AScustomWorks/AS/commit/abd5b5cde20f21f4ba6e1cb78d6d8097e96471dc)

**Comment by AScustomWorks** [comment]
Hi, I made another update.
Some graphics and all the tooltips on buttons, switches and ports are now included on [V2.0.2](https://github.com/AScustomWorks/AS/commit/771e499a71e57c22760f80c7a787642e71f8c195)
Edited (again, sorry) to reflect the last commit(readme.md update with proper description, and a typo fix)

**Comment by AScustomWorks** [comment]
Hi, I made another update:
Update to [V2.0.3](https://github.com/AScustomWorks/AS/commit/9ad1350871e574ff52aca0f02a01bbcc4f56e5a5)
Removed some leftover extra code on VCA and QUADVCA modules that caused wrong behavior when using the CV Level ports.
Thanks!

**Comment by AScustomWorks** [comment]
Hi again, commented out some code by mistake on the V 2.0.3 update for the VCA and QUADVCA, all fixed now. Please update to[ V.2.04](https://github.com/AScustomWorks/AS/commit/fd235b2125685fa514fa8c9f26538b990504c5dd)
Thanks!

**Comment by AScustomWorks** [comment]
Hi!, just a small update with corrected links to the changelog on plugin.json file and graphics license update to CC BY-NC-ND 4.0.
Please update to [V2.0.5](https://github.com/AScustomWorks/AS/commit/2284e4a3befb097b42ae30032147fd61226f64ff)
Thanks!

**Comment by AScustomWorks** [comment]
Hi again, another small update:
Fixed ReScale input 0/5v to output 0/10v range on setting #4
Fixed link to paid plugins on readme.md

Please update to [V2.0.6](https://github.com/AScustomWorks/AS/commit/d90eed32afe6745454ef6ac6e391e0d5dc90ec4b)
Thanks!
